opencti-platform/opencti-front/src/private/components/data/playbooks/Playbook.jsx

```diff
@@ -105,10 +105,9 @@ const PlaybookComponent = ({ playbook, playbookComponents }) => {
       />
       <PlaybookHeader playbook={playbook} />
       <ErrorBoundary
-        display={
-          <div style={{ paddingTop: 28 }}>
-            <SimpleError />
-          </div>
+        display={(errorData) => <div style={{ paddingTop: 28 }}>
+          <SimpleError errorData={errorData}/>
+        </div>
```

Comment:
Each time we call `SimpleError` we encapsulate it with this paddingTop: 28px. Why not putting this style inside the component itself and avoid duplicating code?